README.md

```diff
@@ -1,23 +1,53 @@
-# .NET Thailand official website (https://www.dotnetthailand.com)
+# .NET Thailand official website
+
+![image](https://user-images.githubusercontent.com/344784/114385147-d33a4b00-9bb9-11eb-8aaa-ef0a3ccf4d96.png)
```

Comment:
Could you please move the image file to `src/images/` folder and rename a file?
Please also add alternative text of the image in [ ].
For example, 
```
![logo-banner](src/images/logo-banner.png)
```